utils/dataset_download.py

```diff
@@ -0,0 +1,19 @@
+import os
+from kaggle.api.kaggle_api_extended import KaggleApi
+
+link = 'https://www.kaggle.com/datasets/parisrohan/credit-score-classification' #just for convinience, not important
+download_path = "./data"
+dataset_name = "parisrohan/credit-score-classification"
+
+def download_dataset():
+    """create 'data' folder if it doesnt exist"""
+    os.makedirs(download_path, exist_ok=True)
```

Comment:
Path must use the file itself, due to different scenarios where the function could be called.
You can use something like:
```
root_dir = os.path.dirname(os.path.dirname(__file__))
data_folder_path = os.path.join(root_dir, 'data')
os.makedirs(download_path, exist_ok=True)
```